src/modules/getMovies.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable linebreak-style */
```

Comment:
Please avoid disabling linters and fix the linter error message as it is good Javascript practice